tasks/add-impl-to-finder.js

```diff
@@ -4,8 +4,16 @@ task("add-impl-to-finder", "Adds assets to Asset Manager")
     .addParam("name", "The name of the interface")
     .addParam("address", "The address of the implementation")
     .addFlag("skipIfImplementationExists", "boolean value to skip changing implementation if it already exists")
+    .addOptionalParam(
+        "gnosisSafe",
+        "Gnosis safe address, should be given if trasactions need to be submitted to gnosis",
+        undefined,
+        types.string
+    )
     .setAction(
         async (args, hre) => {
+            if (args.gnosisSafe && !ethers.utils.isAddress(args.gnosisSafe)) throw new Error("Invalid safe address")
```

Comment:
we can do this check on line 36 itself. No need to do it once here and then again on line 36..

Reply:
cleaned it